README.md

```diff
@@ -61,53 +62,129 @@ Versioning is not supported at this time.
 
 # Setting it up <a id="Interesting_Part"/>
 
-## Compilation
+## Building
+
+This project relies on iRODS features that have not been released yet.
+
+You'll need to build the iRODS packages using irods/irods@6b14b65 or a later commit.
+
+Building requires the following:
+- Docker
+- irods-dev
+- irods-runtime
+
+You'll need to copy the packages into the root of the project directory.
```

Comment:
I thought the README would highlight the irods/irods@6b14b65, but it isn't for some reason.

Perhaps it's because it's in a PR?

Reply:
Fixed the linking issue.